models/constant-baseline/Rupdate-constant-baseline.R

```diff
@@ -0,0 +1,117 @@
+if (baseline == "constant baseline") {
+  # get last observed value for median 
+  
```

Comment:
file name should no include name of model or R. its in the constant-baseline folder so this is repetition.

Reply:
please update name to be `update.R` or similar having `constant-baseline` in both folder and filename makes me sad.

Reply:
I don't want to make you sad! :(